core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/ColumnFormatRelation.scala

```diff
@@ -179,7 +181,6 @@ abstract class BaseColumnFormatRelation(
       Seq[RDD[InternalRow]]) = {
     val rdd = scanTable(table, requiredColumns, filters, -1)
     val partitionEvaluator = rdd match {
-      case c: ColumnarStorePartitionedRDD => c.getPartitionEvaluator
       case r => () => r.partitions
     }
```

Comment:
Can we remove partitionEvaluator and replace it with "() => rdd.partitions" in usage?